mule-migration-tool-library/src/test/java/com/mulesoft/tools/migration/library/mule/steps/pom/UpdateProjectParentTest.java

```diff
@@ -0,0 +1,109 @@
+
+  private static final String POM = "/pommodel/simple-pom/pom.xml";
+  private static final String PARENT_POM = "/pommodel/simple-pom/parent-pom.xml";
+
+  @Rule
+  public ReportVerification report = new ReportVerification();
+
+
+  @Test
+  public void executeWithoutProjectParentGavAndWithoutParent() throws IOException, XmlPullParserException, URISyntaxException {
+    UpdateProjectParent updateProjectParent = new UpdateProjectParent();
+    final ApplicationModel applicationModelMock = mock(ApplicationModel.class);
+    when(applicationModelMock.getProjectParentGAV()).thenReturn(Optional.ofNullable(null));
+    updateProjectParent.setApplicationModel(applicationModelMock);
+
+    Path pomPath = Paths.get(getClass().getResource(POM).toURI());
+    PomModel model = new PomModel.PomModelBuilder().withPom(pomPath).build();
+
+    try {
+      updateProjectParent.execute(model, report.getReport());
+    } catch (RuntimeException e) {
+      fail("no exception have to be thrown");
+    }
+  }
+
+  @Test
+  public void executeWithoutProjectParentGavAndWithParent() throws URISyntaxException, IOException, XmlPullParserException {
+    UpdateProjectParent updateProjectParent = new UpdateProjectParent();
+    final ApplicationModel applicationModelMock = mock(ApplicationModel.class);
+    when(applicationModelMock.getProjectParentGAV()).thenReturn(Optional.ofNullable(null));
+    updateProjectParent.setApplicationModel(applicationModelMock);
+
+    Path pomPath = Paths.get(getClass().getResource(PARENT_POM).toURI());
+    PomModel model = new PomModel.PomModelBuilder().withPom(pomPath).build();
+    try {
+      updateProjectParent.execute(model, report.getReport());
+    } catch (RuntimeException e) {
+      fail("no exception have to be thrown");
+    }
+  }
+
+  @Test
+  public void executeWithProjectParentGavAndWithoutParent() throws URISyntaxException, IOException, XmlPullParserException {
+    UpdateProjectParent updateProjectParent = new UpdateProjectParent();
+    final ApplicationModel applicationModelMock = mock(ApplicationModel.class);
+    when(applicationModelMock.getProjectParentGAV()).thenReturn(Optional.ofNullable("com.mulesoft:parent:2.0.0"));
+    updateProjectParent.setApplicationModel(applicationModelMock);
+
+    Path pomPath = Paths.get(getClass().getResource(POM).toURI());
+    PomModel model = new PomModel.PomModelBuilder().withPom(pomPath).build();
+    try {
+      updateProjectParent.execute(model, report.getReport());
+    } catch (RuntimeException e) {
+      fail("no exception have to be thrown");
+    }
+  }
+
+  @Test
+  public void executeWithGavAndParent() throws URISyntaxException, IOException, XmlPullParserException {
+    UpdateProjectParent updateProjectParent = new UpdateProjectParent();
+    final ApplicationModel applicationModelMock = mock(ApplicationModel.class);
+    when(applicationModelMock.getProjectParentGAV()).thenReturn(Optional.ofNullable("com.mule:rest-parent:2.0.0"));
```

Comment:
replace _Optional.ofNullable("com.mulesoft:parent:2.0.0")_ by _Optional.of("com.mule:rest-parent:2.0.0")_

Reply:
done